lib/algoliasearch-rails.rb

```diff
@@ -560,62 +507,80 @@ def algolia_reindex(batch_size = AlgoliaSearch::IndexSettings::DEFAULT_BATCH_SIZ
             group = group.select { |o| algolia_indexable?(o, tmp_options) }
           end
           objects = group.map { |o| tmp_settings.get_attributes(o).merge 'objectID' => algolia_object_id_of(o, tmp_options) }
-          tmp_index.save_objects(objects)
+
+          AlgoliaSearch.client.save_objects(tmp_index_name, objects)
```

Comment:
ah there is no way to override `wait_for_tasks` currently, is this something needed here ?

Reply:
`wait_for_tasks` right after we do the `save_objects` here you mean? I didn't add it because the previous version didn't wait here as well, it shouldn't cause any issues since we're using the same instance of the client so operations should go to the same machine. Open to adding `wait_for_task` here though if you and others think that would be useful!

Reply:
if it was like that before it's fine, just a future improvement then